file/writable_file_writer.h

```diff
@@ -143,6 +143,7 @@ class WritableFileWriter {
   // Actually written data size can be used for truncate
   // not counting padding data
   std::atomic<uint64_t> filesize_;
+  std::atomic<uint64_t> flushed_size_;
```

Comment:
A comment describing `flushed_size_` would be nice, mentioning this size will be synced when calling `SyncWAL()`, etc.

Reply:
Comment added to `GetFlushedSize()`